### PR TITLE
maintainers file: add missing labels

### PR DESCRIPTION
**Pull request by nashif**
- also fix Kconfig area to not include every single Kconfig file.

### CODE REVIEW

File: MAINTAINERS.yml [hunk 1]
```diff
@@ -793,6 +826,9 @@ Little FS:
     description: >-
        Little FS
 
+    labels:
+        - "area: File System"
```
Comment (ioannisg) [inline MAINTAINERS.yml:830]: Could this be specific to "Little FS"

**Comment by galak** [comment]
Is there anything yet parsing MAINTAINERs?

**Review by ioannisg — APPROVED** [review]
thanks

**Review by galak — APPROVED** [review]

File: MAINTAINERS.yml (hunk 1 above)
Comment (nashif) [reply MAINTAINERS.yml:830]: IMO this is going to be too specific, if we start adding labels for each implementation.

**Comment by nashif** [comment]
> Is there anything yet parsing MAINTAINERS?

`./scripts/get_maintainer.py areas`


Note that there is in progress work to change the schema slightly and also change the way we parse the file in preparation for adding GH action

File: MAINTAINERS.yml (hunk 1 above)
Comment (de-nordic) [reply MAINTAINERS.yml:830]: @ioannisg I agree with @nashif. I have one comment though, there is "vfs" which is also listed under "FIle System" area and it contains driver for FAT file system; if the LittleFS got its own section, then maybe the FAT FS should also get one.
VFS itself should contain file system API common files that are not file system specific, IMO.

**Review by MaureenHelm — APPROVED** [review]

File: MAINTAINERS.yml (hunk 1 above)
Comment (nashif) [reply MAINTAINERS.yml:830]: > there is "vfs" which is also listed under "FIle System" area and it contains driver for FAT file system;

yes, I think we can move FAT into its own entry

**Pull request merged**